jobs/JHAFS_ANALYSIS

```diff
@@ -19,6 +19,7 @@ export RUN_ENVIR=${RUN_ENVIR:-dev} # nco or dev
 if [ "${RUN_ENVIR^^}" != NCO ]; then
   module use $HOMEhafs/sorc/hafs_gsi.fd/modulefiles
   if [ ${machine}  = "hera" ]; then
+    module purge
```

Comment:
@panll, for this fix, not sure if it is related to your own module environment. Our EMC and AOML colleagues seem to run HAFS fine on Hera without this fix.

Reply:
That's possible. I'll remove this part. Thanks! @BinLiu-NOAA